MANIFEST.in

```diff
@@ -1,8 +1,23 @@
-#include *.txt
+include AUTHORS.md
+include LICENSE
+include README.md
+
 include versioneer.py
 include pvlib/_version.py
+
 recursive-include pvlib/data *
-include README.md
+
 include pvlib/spa_c_files/*.h
```

Comment:
IMO this should be removed, b/c I don't think there are any header files are in `spa_c_files` that should be distributed, and this might accidentally cause the NREL SPA C-files to be distributed without their consent.